activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,37 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Applies an +ORDER BY+ clause to a query.
```

Comment:
Sorry, this is my fault -- this needs to be

```suggestion
    # Applies an <code>ORDER BY</code> clause to a query.
```

Because of the space:

```console
> require "rdoc"
> RDoc::Markup::ToHtml.new(RDoc::Options.new).convert("+ORDER BY+ vs +ORDERBY+")
=> "\n<p>+ORDER BY+ vs <code>ORDERBY</code></p>\n"
```

Reply:
Looking at some similar methods in this file, I'm not sure we should use `ORDER BY` here.
None of the other examples are using SQL in the description.

It also seems we use mostly `relation` instead of `query`.
This makes sense I guess, because a relation is returned.

`reverse_order`
https://github.com/rails/rails/blob/8807b93841ddd1a7fa2b2f44f02e1f690f3d9cc9/activerecord/lib/active_record/relation/query_methods.rb#L1151
`reorder`
https://github.com/rails/rails/blob/8807b93841ddd1a7fa2b2f44f02e1f690f3d9cc9/activerecord/lib/active_record/relation/query_methods.rb#L440
`limit`
https://github.com/rails/rails/blob/8807b93841ddd1a7fa2b2f44f02e1f690f3d9cc9/activerecord/lib/active_record/relation/query_methods.rb#L876
`Where`
https://github.com/rails/rails/blob/8807b93841ddd1a7fa2b2f44f02e1f690f3d9cc9/activerecord/lib/active_record/relation/query_methods.rb#L590-L591

```suggestion
    # Applies an order to the relation.
```

Or maybe, since multiple orders can be added:

```suggestion
    # Adds an order to the relation.
```

Sorry for the bikeshed  🙇

Reply:
> Looking at some similar methods in this file, I'm not sure we should use `ORDER BY` here.
> None of the other examples are using SQL in the description.

There are a few that do:

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activerecord/lib/active_record/relation/query_methods.rb#L20-L21

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activerecord/lib/active_record/relation/query_methods.rb#L185

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activerecord/lib/active_record/relation/query_methods.rb#L258-L259

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activerecord/lib/active_record/relation/query_methods.rb#L390-L391

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activerecord/lib/active_record/relation/query_methods.rb#L830-L831

That said, I don't have a strong preference. :smile:

Reply:
Indeed 👍. I’ll leave it up to @ChaelCodes

Reply:
I think including the ORDER BY provides more information than just saying `order` but I'm 50/50 on query vs relation, so I'll replace it with relation next time I'm at my computer.

Reply:
If we are sticking with the `ORDER BY` I guess it makes sense to also stick with query 🙂 .